addons/survey/views/survey_templates_statistics.xml

```diff
@@ -215,16 +239,36 @@
                     <thead>
                         <tr>
                             <th>User Responses</th>
-                            <th>Occurrence</th>
+                            <!-- todo: decide if we add   t-if="question.question_type != 'datetime' or question.is_scored_question" to th-->
+                            <th class="col-2">Occurrence</th>
+                            <th t-if="question.is_scored_question" class="col-2">Score</th>
                         </tr>
                     </thead>
                     <tbody>
                         <tr t-foreach="question_data['common_lines']" t-as="common_line">
                             <td>
-                                <span t-if="question.question_type == 'numerical_box'" t-esc="common_line[0]"/>
-                                <span t-if="question.question_type == 'date'" t-esc="common_line[0]" t-options='{"widget": "date"}'/>
+                                <t t-if="question.question_type == 'numerical_box'">
+                                    <t t-set="right_answer" t-value="question.is_scored_question and common_line[0] == question.answer_numerical_box"/>
+                                    <span><t t-call="survey.survey_format_if_integer"><t t-set="value_to_format" t-value="common_line[0]"/></t></span>
+                                    <i t-if="right_answer" class="fa fa-check"/>
+                                </t>
+                                <t t-elif="question.question_type == 'date'">
+                                    <t t-set="right_answer" t-value="question.is_scored_question and common_line[0] == question.answer_date"/>
+                                    <span t-esc="common_line[0]"  t-options='{"widget": "date"}'></span>
+                                    <i t-if="right_answer" class="fa fa-check"/>
+                                </t>
+                                <t t-elif="question.question_type == 'datetime'">
+                                    <t t-set="right_answer" t-value="question.is_scored_question and common_line[0] == question.answer_datetime"/>
+                                    <span t-esc="common_line[0]"  t-options='{"widget": "datetime"}'></span>
+                                    <i t-if="right_answer" class="fa fa-check"/>
+                                </t>
                             </td>
+                            <!-- todo: decide if we add   t-if="question.question_type != 'datetime' or question.is_scored_question" to td-->
                             <td><span t-esc="common_line[1]"></span></td>
+                            <td t-if="question.is_scored_question">
+                                <span t-if="right_answer"><t t-call="survey.survey_format_if_integer"><t t-set="value_to_format" t-value="question.answer_score"/></t></span>
```

Comment:
It's generally a good idea to indent more to avoid long lines.
It gets hard to read pretty fast on a regular screen.

So:
```xml
<span t-if="right_answer">
    <t t-call="survey.survey_format_if_integer">
        <t t-set="value_to_format" t-value="question.answer_score"/>
    </t>
</span>
```

Also, I think it works if you embed the t-call within your span, to test:

```xml
<span t-if="right_answer" t-call="survey.survey_format_if_integer">
    <t t-set="value_to_format" t-value="question.answer_score"/>
</span>
```

(Please apply the same to other occurrences as well).